manifests/resolved.pp

```diff
@@ -176,6 +180,23 @@
     }
   }
 
+  $_dnsovertls = $dnsovertls ? {
+    true    => 'opportunistic',
+    false   => 'no',
+    default => $dnsovertls,
+  }
+
+  if $_dnsovertls {
```

Comment:
can you please have a look at the `spec/classes/init_spec.rb` file add add some spec tests for this change?

Reply:
sure